float64[] initialState
quadruped_control/Pose goal
---
float64[] solution
int32 result
string error_message